app/models/edition/custom_lead_image.rb

```diff
@@ -0,0 +1,37 @@
+module Edition::CustomLeadImage
+  extend ActiveSupport::Concern
+
+  included do
+    validates :body,
+              format: {
+                without: /^!!1([^\d]|$)/,
```

Comment:
I'd suggest it should probably be included in this one? Otherwise I feel that change will end up being largely unassociated with the rest of the next commit. But if that's not the case and you feel it fits better in the next PR then go for it

Reply:
Just asked about this in slack. I think we will need to backfill the lead image association before updating this regex to use the new association. Maybe i should handle renaming this module and updating it in a separate PR after this one has been merged and backfilled? 

Either that or i'll handle updating this as part of the next PR which updates all the code that uses `images.first` to `edition.lead_image`. Let me know if you've got a preference.